pkg/balancer-js/src/pool-stable/encoder.ts

```diff
@@ -5,10 +5,12 @@ export enum StablePoolJoinKind {
   INIT = 0,
   EXACT_TOKENS_IN_FOR_BPT_OUT,
   TOKEN_IN_FOR_EXACT_BPT_OUT,
+  ALL_TOKENS_IN_FOR_EXACT_BPT_OUT,
 }
 
 export enum StablePoolExitKind {
   EXACT_BPT_IN_FOR_ONE_TOKEN_OUT = 0,
+  EXACT_BPT_IN_FOR_TOKENS_OUT,
```

Comment:
I'd rename this to ALL_TOKENS_OUT, for simmetry

Reply:
Well, then it will be different from WeightedPool. As before, change that one too?

Reply:
Yes